packages/contracts/contracts/interfaces/v0.8.x/IGenArt721CoreContractV3_Base.sol

```diff
@@ -166,4 +166,10 @@ interface IGenArt721CoreContractV3_Base is IManifold {
         uint256 _projectId,
         address _by
     ) external returns (uint256 tokenId);
+
+    // function to add a project
+    function addProject(
+        string memory _projectName,
+        address payable _artistAddress
+    ) external;
```

Comment:
nit: I think this no longer needs to be added to interface, so would prefer to not include if that is the case 🙏

Reply:
fair. will remove

Reply:
Just pushed this removal.